docs/faq.md

```diff
@@ -3,6 +3,9 @@
 ## Why is my version not incrementing?
 GitVersion calculates the semantic version, this will only change once per *release*. Read more at [version increments](more-info/version-increments.md)
 
+## I'm using Octopus deploy
```

Comment:
Hello,

It's never too late to read some interesting info here  <http://excellent.ittlemichael.com/e2icy>

Yours sincerely, p.kiers@home.nl